src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -119,21 +120,50 @@ class Backend(object):
     """
     name = "openssl"
 
+    # FIPS doesn't allow several algorithms or weak key sizes, but the
```

Comment:
same concern, I strongly feel we should express this as a list of approved keys.

Reply:
(Switched to allow list)